src/chi-vue/src/components/data-table-filters/DataTableFilters.tsx

```diff
@@ -70,12 +72,15 @@ export default class DataTableFilters extends Vue {
 
     this._advancedFiltersData = copyArrayOfObjects(advancedFilters);
 
-    const plainFiltersData = this.$props.filtersData.reduce((accumulator: any, currentValue: any) => {
-      return { ...accumulator, [currentValue.id]: currentValue.type === 'checkbox' ? false : currentValue.value || '' };
-    }, {});
+    this.$props.filtersData.forEach((currentValue: any) => {
```

Comment:
Is there a better type here?
```suggestion
    this.$props.filtersData.forEach((currentValue: any) => {
```